sdk/storage/azblob/container/client_test.go

```diff
@@ -2371,28 +2371,13 @@ func (s *ContainerUnrecordedTestsSuite) TestContainerBlobBatchSetTierPartialFail
 	_require.Equal(ctrCool, 10)
 }
 
-/*
 func (s *ContainerUnrecordedTestsSuite) TestContainerBlobBatchDeleteUsingTokenCredential() {
 	_require := require.New(s.T())
 	testName := s.T().Name()
-	accountName, ok := os.LookupEnv("AZURE_STORAGE_ACCOUNT_NAME")
-	if !ok {
-		panic("AZURE_STORAGE_ACCOUNT_NAME could not be found")
-	}
-	tenantID, ok := os.LookupEnv("AZURE_STORAGE_TENANT_ID")
-	if !ok {
-		panic("AZURE_STORAGE_TENANT_ID could not be found")
-	}
-	clientID, ok := os.LookupEnv("AZURE_STORAGE_CLIENT_ID")
-	if !ok {
-		panic("AZURE_STORAGE_CLIENT_ID could not be found")
-	}
-	clientSecret, ok := os.LookupEnv("AZURE_STORAGE_CLIENT_SECRET")
-	if !ok {
-		panic("AZURE_STORAGE_CLIENT_SECRET could not be found")
-	}
+	accountName, _ := testcommon.GetGenericAccountInfo(testcommon.TestAccountDefault)
+	_require.Greater(len(accountName), 0)
 
-	cred, err := azidentity.NewClientSecretCredential(tenantID, clientID, clientSecret, nil)
+	cred, err := testcommon.GetGenericTokenCredential()
```

Comment:
are we able to record these now with GetGenericTokenCred?

Reply:
All tests for blob batch are unrecorded currently whether we use shared key or OAuth credential. This is because in blob batch a random uuid is passed in the [Content-Type](https://learn.microsoft.com/en-us/rest/api/storageservices/blob-batch#request-headers) request header. For example, multipart/mixed; boundary=batch_**a81786c8-e301-4e42-a729-a32ca24ae252**.
This uuid is also passed in the request body as a batch boundary. I did try to add sanitizers for both the header and the body. But it didn't seem to resolve this. So, I have these tests as unrecorded.